eng/versioning/external_dependencies.txt

```diff
@@ -234,6 +234,7 @@ cosmos_io.dropwizard.metrics:metrics-core;4.1.0
 cosmos_io.dropwizard.metrics:metrics-graphite;4.1.0
 cosmos_io.dropwizard.metrics:metrics-jvm;4.1.0
 cosmos_org.mockito:mockito-core;1.10.19
+cosmos_org.bouncycastle:bcprov-ext-jdk15on;1.63
```

Comment:
Please do not add this as a cosmos specific dependency. Those are only supposed to be used if there's a conflict with a common dependency which can't be clearly updated. In this particular case org.bouncycastle:bcprov-ext-jdk15on is brand new so it should be up with the others.

Reply:
understood, thanks @JimSuplizio will change.

Reply:
new dependency removed switched to the existing dependency which suffices for our case 
https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/external_dependencies.txt#L127